visualization/app/codeCharta/util/gameObjectsParser/gameObjectsImporter.ts

```diff
@@ -0,0 +1,138 @@
+import md5 from "md5"
+
+function isFile(names) {
+	return names.length === 0
+}
+
+function nodeAlreadyExists(nodes, name) {
+	return nodes.some(node => node.name === name)
+}
+
+function addNodeRecursively(names, nodes, gameObjectPosition, parentNodeName, gameObjectPositions) {
+	if (names.length === 0) return
+	// get current path name
+	const name = names.shift()
+	let node
+
+	// create node
+	node = { name, type: isFile(names) ? "File" : "Folder", attributes: {} }
```

Comment:
```suggestion
	// create node
	let node = { name, type: isFile(names) ? "File" : "Folder", attributes: {} }
```

Reply:
If I declare it there. I need to turn node.children to node["children"]. Is that a bad practice?

Reply:
I found another way
```suggestion
	// create node
	let node = { name, type: isFile(names) ? "File" : "Folder", attributes: {}, children: undefined}
```
Then I can assign children if it's needed (in case of a folder) let me know if this is not good practice.